app/testOnly/views/TestOnlyStartPage.scala.html

```diff
@@ -197,12 +219,11 @@
 
 @testOnlyLayout(pageTitle = Some(title)) {
     @if(form.errors.nonEmpty) {
-        @govukErrorSummary(ErrorSummary(errorList = form.errors.map(e =>
-            ErrorLink(
-                href = Some(s"#${e.key}"),
-                content = Text(e.message)
-            )
-        ),
+        @govukErrorSummary(ErrorSummary(errorList = form.errors.map { e =>
+            StartJourneyController.totalDebtErrorMessage(e.message) match {
```

Comment:
It does for enums, but this is the first form component that isn't an enum, so this made it work (it's only actually for the error message at the very top of the page, otherwise it said error.required/error.tooSmall etc, which admittedly isn't a deal breaker for the test only page)

Reply:
Maybe I just remove the maximum constraint? it is only test only

Reply:
would it work to leave it as it was and do a mapping of the `FormError` just before passing it to the template? Something like this in the controller submit method:
```scala
        formWithErrors => {
          import cats.syntax.eq._
          val form = formWithErrors.copy(errors = formWithErrors.errors.map( e =>
          if(e.key === "debtTotalAmount") {
            e.withMessage(StartJourneyController.amountInputErrorMessage(e.message)
              .getOrElse(sys.error(s"Could not find error message for '${e.message}' for debtTotalAmount "))
            )
          } else e
          ))
          Future.successful(Ok(testOnlyStartPage(form)))
```

Reply:
Worked a treat, thanks!